docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -336,6 +336,9 @@ In a playbook, you can control the collections Ansible searches for modules and
 
 The ``collections`` keyword merely creates an ordered 'search path' for non-namespaced plugin and role references. It does not install content or otherwise change Ansible's behavior around the loading of plugins or roles. Note that an FQCN is still required for non-action or module plugins (for example, lookups, filters, tests).
 
+When using the ``collections`` variable, it is not necessary to add in ``ansible.builtin`` as part of the search list. When left omitted, ``ansible.legacy``, a superset of ``ansible.builtin``, is auto-appended to the list of available collections.
```

Comment:
Rather than call `ansible.legacy` a superset of `ansible.builtin` in user docs, it's probably clearer to explain what each synthetic collection contains (it doesn't seem like we really have any documentation on what they are or when they were added; there's another recent issue that assumed initially they were 2.10 features). `ansible.builtin` is content in `ansible-base`/`ansible-core` and `ansible.legacy` is that plus old-style 3rd party plugin paths (like https://github.com/ansible/ansible/blob/devel/examples/ansible.cfg#L199).

Reply:
Yeah, I agree in that it wouldn't clarify much to the average user

How about:

```
When using the ``collections`` variable, it is not necessary to add in ``ansible.builtin`` as part of the search list. When left omitted, the following content is available by default:

1. Standard ansible modules and plugins available through ``ansible-base/ansible-core``
2. Support for older 3rd party plugin paths
```
Is that more along the lines of what you were looking for? Also, is there anything else that should explicitly to be a part of that list?

Reply:
@syedahaider97 That seems good to me. Maybe just ``ansible-core`` for devel and if backported, that would be changed to ``ansible-base`` (core content was renamed for 2.11). Someone from the docs team may want to do some wordsmithing - @acozine, do you have any thoughts on this?

Reply:
I'll wait on their input before revising the PR :)